tests/core/test_stack_align.cpp

```diff
@@ -9,6 +9,7 @@ struct Aligned {
   char ALIGN(32) a32;
 };
 
+__attribute__((noinline))
```

Comment:
Haven't fixed this one yet. Current working theory is that LLVM backend isn't reconciling the alignment of the struct with the alignment of the stack pointer (SP is 16byte aligned, struct is 32byte aligned, so we need to recognize that we need a frame pointer that's 32byte aligned).

The change here is just a more precise way to specify the behavior that the test seems to rely on, i.e. that `Test` isn't inlined.